src/api/newComment.ts

```diff
@@ -0,0 +1,6 @@
+import { Comment } from '../types/Comment';
+import { client } from '../utils/fetchClient';
+
+export const newComment = (comment: Partial<Comment>) => {
+  return client.post('/comments', comment);
```

Comment:
Consider adding error handling for the `newComment` API call. Including a `.catch` block will help manage errors during the request and provide feedback to the user if the comment creation fails.